### ISSUE TITLE
Seems a shame not to also include a FileField

### ISSUE THREAD

**Issue by andybak** [bradleyg/django-ajaximage#15]
As I presume a lot of the code would be similar?

Ideally - the field could still display a thumbnail if the file was a recognised file format...

**Comment by jimmykobe1171** [comment]
Yes, I assume it to extend the FileField class, but it just inherits the Field class. Furthermore, AjaxImageField misses many methods in FileField, which may cause problems.
